Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiBase.cs

```diff
@@ -73,6 +75,7 @@ public void GenerateResponse(SwaggerOperation operation, ApplicationAPIModel bas
                             arv.DoNotConsiderAsTemp = true;
                             basicModal.ReturnValues.Add(arv);
                         }
+                        var test = operation.RequestBody.Content;
```

Comment:
Please remove this test variable if not being used

Reply:
Not in use, removed.